dev-requirements.txt

```diff
@@ -0,0 +1,15 @@
+###############################################################################
+# The runtime dependencies are limited to only the QGIS environment. These dev
+# dependencies only provide support for non-runtime behaviors, e.g.:
+# building docs locally, bumping versions, static analysis, etc.
+###############################################################################
+
+# Documentation:
+sphinx ~=4.3.2
+sphinx-rtd-theme ~=1.0.0
+myst-parser ~=0.15.2
```

Comment:
These are all included on readthedocs, so we don't need to specify a requirements.txt file in the `help` directory anymore.

Reply:
Just kidding, apparently `myst_parser` is not.

Reply:
8bd3e09